processor/schemaprocessor/README.md

```diff
@@ -0,0 +1,35 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Processor_ is used to convert existing telemetry data or signals to a version of the semantic convention defined as part of the configuration.
+
+
+Specification Reference: [here](https://opentelemetry.io/docs/reference/specification/schemas/overview/)
+
+## Caching Schema Translation Files
+
+In order to improve effeicency of the processor, the `precache` option allows the processor to start downloading and preparing
+the translations needed for signals that match the schema url.
+
+## Targets Schemas
+
+Targets define a list of schema urls with a schema identifer that will be used to translate any schema url that matches the target url to that version.
+In the event that the processor matches a signal to a target, the processor will translate the signal from published one to the defined identifier;
+using the configuration bellow, a signal published with `SchemaURL:https://opentelemetry.io/schames/1.8.2` will be translated by the collector to `SchemaURL:https//opentelemetry.io/schemas/1.6.2` so that all existing visualisations will remain unchanged.
+
+Furthemore, it is also possible for organisations and vendors to publish their own semantic convention and be used by this processor, be sure to follow [schema overview](https://opentelemetry.io/docs/reference/specification/schemas/overview/) for all the details.
+# Example
+
+```yaml
+processors:
+    schema_transformer:
+        precache:
+        - https://opentelemetry.io/schemas/1.9.0
+        targets:
+        - https://opentelemetry.io/schemas/1.6.2
```

Comment:
Nit: let's use URLs that actually exist:
```suggestion
        - https://opentelemetry.io/schemas/1.6.1
```

Reply:
Question for you, is there an easier way for a developer / curious person to see the list of published schemas?

Reply:
Yes, all Otel schema file originals are here: https://github.com/open-telemetry/opentelemetry-specification/tree/main/schemas